api/users/serializers.py

```diff
@@ -444,6 +449,20 @@ def verify_two_factor(self, instance, value, two_factor_addon):
             raise exceptions.PermissionDenied(detail='The two-factor verification code you provided is invalid.')
         two_factor_addon.save()
 
+    def request_deactivation(self, instance, requested_deactivation):
+        if instance.requested_deactivation != requested_deactivation:
+            if requested_deactivation:
+                mails.send_mail(
+                    to_addr=OSF_SUPPORT_EMAIL,
+                    mail=mails.REQUEST_DEACTIVATION,
+                    user=instance,
+                    can_change_preferences=False,
+                )
+                instance.email_last_sent = timezone.now()
+            instance.requested_deactivation = requested_deactivation
```

Comment:
Probably out of scope for this ticket, but I was wondering -- what's the notification mechanism for "reversing" this deactivation decision? Is OSF Support notified, or is this just a case that'll be handled internally? I just forsee a case if you can request a deactivation reversal from the frontend - a user requests deactivation, reverses it, but OSF support just sees one email and then deactivates it even tho it's been reversed. Maybe this is being paranoid!

Reply:
Good question! Maybe we send a different email - do you have ideas on how you want to handle this @brianjgeiger ?

Reply:
There's a button in the admin's user view that is active if requested_deactivation is true and disabled otherwise. Although she hasn't before, Becca will check that before emailing users just in case.

Reply:
Oh, and an additional email would not be helpful in this case.

Reply:
sounds good! thanks for the info 👍